src/components/auth/SignUpScreen.js

```diff
@@ -1,290 +1,303 @@
-import React from 'react'
-
+import React from "react";
 import {
   TouchableOpacity,
   TouchableWithoutFeedback,
   StyleSheet,
   Text,
+  Dimensions,
   SafeAreaView,
   KeyboardAvoidingView,
   Keyboard,
   View,
   Alert,
   ActivityIndicator,
-} from 'react-native'
-
-import {
-  Container,
-  Item,
-  Input} from 'native-base'
-
-import { Ionicons } from '@expo/vector-icons';
-
-import COLORS from '../../assets/Colors.js'
-
-import Auth from '@aws-amplify/auth'
-
-import { API } from 'aws-amplify'
-
-import i18n from 'i18n-js'
+} from "react-native";
+import { scale, verticalScale, moderateScale } from "react-native-size-matters";
+import { Form, Item, Input, Label } from "native-base";
+import { Ionicons } from "@expo/vector-icons";
+import COLORS from "../../assets/Colors.js";
+import Auth from "@aws-amplify/auth";
+import MenuBar from "../main/MenuBar.js";
+import { API } from "aws-amplify";
+import i18n from "i18n-js";
 
 export default class SignUpScreen extends React.Component {
   state = {
-    userSub: '',
-    name: '',
-    email: '',
-    password: '',
-    password_confirmation: '',
+    userSub: "",
+    name: "",
+    email: "",
+    password: "",
+    password_confirmation: "",
     isLoading: false,
-    hidePassword1: true,
-    hidePassword2: true,
+    hidePassword: true,
     apiResponse: null,
-    UserId: ''
-  }
- 
+    UserId: "",
+  };
+
   handleChangeUserId = (event) => {
-      this.setState({UserId: event});
-  }
-  
+    this.setState({ UserId: event });
+  };
+
   onChangeText(key, value) {
     this.setState({
-      [key]: value
-    })
-  }
-
-  // toggles secure text password 
-  handleHidePassword1 = () => {
-    if (this.state.hidePassword1) {
-      this.setState({hidePassword1: false})
-    } else {
-      this.setState({hidePassword1: true})
-    }
+      [key]: value,
+    });
   }
 
-  // toggles secure text password confirmation
-  handleHidePassword2 = () => {
-    if (this.state.hidePassword2) {
-      this.setState({hidePassword2: false})
+  // toggles secure text password
+  handleHidePassword = () => {
+    if (this.state.hidePassword) {
+      this.setState({ hidePassword: false });
     } else {
-      this.setState({hidePassword2: true})
+      this.setState({ hidePassword: true });
```

Comment:
While I do understand having one toggle control both passwords is ever so slightly easier for the user, I do feel like having a separate one for each looks more balanced.

Reply:
I disagree. The UI is for the user. Clicking two buttons to clarify whether passwords match or are correct can be frustrating and offers more functionality, but for 0 added benefit.

Reply:
Well the purpose of the buttons are to ensure you typed your password correctly, not whether they match. In reality, they could even forgo the toggle and just be a "hold button to show" type thing. As such, having 1 button sacrifices security.

Reply:
Although I see your point, 1 button does not compromise security if that were the case then not having a button at all and keeping the password hidden all the time would be the solution. In addition, a password typed out correctly is equivalent to checking if the passwords match. 1 button, covers both use cases.

Reply:
Okay, for now keep it as is, we can bring this up to Flavio later on during QA period as it's not a hard change in either direction